src/bin/cql-stress-cassandra-stress/settings/param/simple_param.rs

```diff
@@ -66,6 +69,9 @@ impl<T: Parsable> ParamImpl for SimpleParam<T> {
             usage += &format!(" (default={default})");
         }
         println!("{:<40} {}", usage, description);
+        if let Some(additional_description) = &self.additional_desc {
+            println!("{:<40}", additional_description)
```

Comment:
What is `{:<40}` needed for? AFAIK it aligns the string to the left, but is there a need to do it?